python/gtsam/tests/test_SfmData.py

```diff
@@ -56,24 +62,58 @@ def test_data(self):
         """Test functions in SfmData"""
         # Create new track with 3 measurements
         i1, i2, i3 = 3,5,6
-        uv_i1 = gtsam.Point2(21.23, 45.64)
+        uv_i1 = Point2(21.23, 45.64)
+        
         # translating along X-axis
-        uv_i2 = gtsam.Point2(45.7, 45.64)
-        uv_i3 = gtsam.Point2(68.35, 45.64)
+        uv_i2 = Point2(45.7, 45.64)
+        uv_i3 = Point2(68.35, 45.64)
+        
         # add measurements and arbitrary point to the track
         measurements = [(i1, uv_i1), (i2, uv_i2), (i3, uv_i3)]
-        pt = gtsam.Point3(1.0, 6.0, 2.0)
-        track2 = gtsam.SfmTrack(pt)
+        pt = Point3(1.0, 6.0, 2.0)
+        track2 = SfmTrack(pt)
         track2.addMeasurement(i1, uv_i1)
         track2.addMeasurement(i2, uv_i2)
         track2.addMeasurement(i3, uv_i3)
         self.data.addTrack(self.tracks)
         self.data.addTrack(track2)
+
         # Number of tracks in SfmData is 2
         self.assertEqual(self.data.numberTracks(), 2)
+        
         # camera idx of first measurement of second track corresponds to i1
         cam_idx, img_measurement = self.data.track(1).measurement(0)
         self.assertEqual(cam_idx, i1)
 
+    def test_Balbianello(self):
```

Comment:
nit: would be nice to add a 1-liner sentence explaining what the test is striving for